Documentation/FAQ/Index.rst

```diff
@@ -41,6 +41,16 @@ Did you configure your solr connection as required?
 
 |
 
+**In which cases do I want to trigger indexing manually?**
+
+- after changing any configuration file.
+- after modifying synonyms, stop words, protected words in TYPO3 Backend -> Search
+
+Moreover by changing core/index configuration you need to reload the core to make the changes become active.
+To reload configuration you can either restart the whole Solr server or simply reload a specific core.
```

Comment:
I think currently the reload is happening automatically when editing synonyms or stopwords, but the part above regarding synonyms and stopwords is just talking about reIndexing. At least for synonyms a reindexing is still required.